src/app/views/sap/create_mission.php

```diff
@@ -15,10 +15,10 @@
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
                     foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
+                        echo "<p class=\"text-error\">$error</p>";
```

Comment:
Why escape quotes? :/

Please use different quotes, it does not hurt the eyes as this one do. :P

This is one of the reasons to not use echo, btw :P